code/__HELPERS/icon_smoothing.dm

```diff
@@ -387,7 +387,7 @@ DEFINE_BITFIELD(smoothing_junction, list(
 								if(!get_smooth_underlay_icon(underlay_appearance, src, turned_adjacency)) //if all else fails, ask our own turf
 									underlay_appearance.icon = DEFAULT_UNDERLAY_ICON
 									underlay_appearance.icon_state = DEFAULT_UNDERLAY_ICON_STATE
-					underlays = list(underlay_appearance)
+					underlays += underlay_appearance
```

Comment:
have you verified this isn't just causing the underlay to just duplicate up?

Reply:
no ill have to look into that

Reply:
no longer does that since it wasnt the solution to what i was facing anyways